core/validatorapi/validatorapi_test.go

```diff
@@ -1509,6 +1509,56 @@ func TestComponent_SubmitSyncCommitteeContributionsVerify(t *testing.T) {
 	<-done
 }
 
+func TestComponent_AggregateSyncCommitteeSelectionsVerify(t *testing.T) {
+	ctx := context.Background()
+
+	val := testutil.RandomValidator(t)
+
+	// Create keys (just use normal keys, not split tbls)
+	pubkey, secret, err := tbls.Keygen()
+	require.NoError(t, err)
+
+	pubShareByKey := map[*bls_sig.PublicKey]*bls_sig.PublicKey{pubkey: pubkey}
+	pk, err := tblsconv.KeyToCore(pubkey)
+	require.NoError(t, err)
+
+	val.Validator.PublicKey, err = tblsconv.KeyToETH2(pubkey)
+	require.NoError(t, err)
+
+	bmock, err := beaconmock.New(beaconmock.WithValidatorSet(beaconmock.ValidatorSet{val.Index: val}))
+	require.NoError(t, err)
+
+	selection := testutil.RandomSyncCommitteeSelection()
```

Comment:
should be atleast two selections since the endpoint accepts multiple selections.